src/jbuild.ml

```diff
@@ -1634,15 +1632,25 @@ module Documentation = struct
       )
 end
 
+module Include_subdirs = struct
+  type t = Unqualified
+
+  let t =
+    enum
+      [ "unqualified", Unqualified
```

Comment:
One thing that seems a bit weird is that there is no syntax for the default case. Would it be useful to have `(include_subdirs no)`?

Reply:
I suppose. It especially makes sense if we plan to one day change the default to `(include_subdirs qualified)`. I added `no` and reworked the documentation